.github/workflows/ci.yml

```diff
@@ -75,3 +75,21 @@ jobs:
       run: |
         yum -y install gcc make
         make REDIS_CFLAGS='-Werror'
+
+  schema-validator:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: make
+        # Fail build if there are warnings
+        run: make REDIS_CFLAGS='-Werror -DLOG_REQ_RES'
+      - name: test
+        run: |
+          sudo apt-get install tcl8.6 tclx
+          ./runtest --verbose --tags -slow --dump-logs --log-req-res --dont-clean --force-resp3
+      - name: Install Python dependencies
+        uses: py-actions/py-dependency-install@v4
+        with:
+          path: "./utils/req-res-validator/requirements.txt"
+      - name: validator
+        run: ./utils/req-res-log-valiadtor.py
```

Comment:
it doesn't run the tests so what does this validator do?
if it just validates the schema is valid, we can probably keep it in ci.yml, but we'll also want one job that runs the tests and validates the replies, that one can initially be put in ci.yml, but will later need to be moved to daily or weekly.

Reply:
yes, it's here just in order to validate the schemas
before merging we will move it to daily

Reply:
if it just validates the schema, and is quick, maybe we can keep it in ci.yml

Reply:
it's not quick (±12 minutes)

Reply:
but this job doesn't run any redis tests. what's taking so long?

Reply:
@oranagra see the line 89. It calls `./runtest ...`.

Reply:
ohh, i'm blind! i was looking for it, and couldn't find it.